source/index.html

```diff
@@ -129,10 +131,213 @@ <h1 class="row-title">Top Dinner Recipes</h1>
+                                    <select name="ordering" id="ordering" value="">
+                                        <option value="">None selected</option>
+                                        <option value="descending">Descending</option>
+                                        <option value="ascending">Ascending</option>
+                                    </select>
+                                </div>
+                            </div>
+                            <hr class="solid">
+                            <div class="filters-meal-type">
+                                <div class="filters-meal-type">
+                                <div class="filter-meal-type-input">
+                                    <label class="sort-filter-label">Meal Type:</label><br>
+                                    <select name="meal-type" id="meal-type" value="">
+                                        <option value="">None selected</option>
+                                        <option value="main course">Main course</option>
+                                        <option value="side dish">Side dish</option>
+                                        <option value="dessert">Dessert</option>
+                                        <option value="appetizer">Appetizer</option>
+                                        <option value="salad">Salad</option>
+                                        <option value="bread">Bread</option>
+                                        <option value="breakfast">Breakfast</option>
+                                        <option value="soup">Soup</option>
+                                        <option value="beverage">Beverage</option>
+                                        <option value="sauce">Sauce</option>
+                                        <option value="marinade">Marinade</option>
+                                        <option value="fingerfood">Fingerfood</option>
+                                        <option value="snack">Snack</option>
+                                        <option value="drink">Drink</option>
+                                    </select>
+                                </div>
+                                </div>
+                            </div>
+                            <hr class="solid">
+                            <div class="filters-cuisine">
+                            <div class="filters-cuisine-header">
+                                <label class="sort-filter-label">Filter by cuisine:</label>
+                                <button class="cuisine-collapse" id="cuisine-collapse" type="button" onclick="displayCuisine()">↑</button>
+                            </div>
+                            <div class="filters-cuisine-body" id="cuisine-collapse-body">
+                                <div class="filter-cuisine-input">
+                                <input type="checkbox" id="cuisine1" name="cuisine1" value="African">
```

Comment:
Why are the ID's cuisine# rather than just the name of the cuisine type? i.e. name = "African" instead of name = "cuisine1"

Reply:
@fjontran